tensorflow_addons/layers/tests/polynomial_test.py

```diff
@@ -31,14 +31,23 @@ def test_full_matrix():
     np.testing.assert_allclose([[0.55, 0.8, 1.05]], output)
 
 
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
+def test_low_rank_matrix():
+    x0 = np.asarray([[0.1, 0.2, 0.3]]).astype(np.float32)
+    x = np.asarray([[0.4, 0.5, 0.6]]).astype(np.float32)
+    layer = PolynomialCrossing(projection_dim=1, kernel_initializer="ones")
+    output = layer([x0, x])
+    np.testing.assert_allclose([[0.55, 0.8, 1.05]], output)
+
+
 @pytest.mark.usefixtures("maybe_run_functions_eagerly")
 def test_invalid_proj_dim():
     with pytest.raises(ValueError) as exception_info:
         x0 = np.random.random((12, 5))
         x = np.random.random((12, 5))
         layer = PolynomialCrossing(projection_dim=6)
         layer([x0, x])
-    assert "is not supported yet" in str(exception_info.value)
+    assert "should be smaller than last_dim / 2" in str(exception_info.value)
```

Comment:
Could you add this into the `matches` parameter in pytest.raises instead of this separate assertion.

Reply:
Good point. Done